pktoken/pktoken.go

```diff
@@ -357,26 +384,18 @@ func (p *PKToken) VerifyCosSig(cosPk jwk.Key, alg jwa.KeyAlgorithm) error {
 	return nil
 }
 
-func (p *PKToken) Verify(msg string, sig []byte) error {
-	jwsSig, err := jws.Parse(sig)
+func (p *PKToken) Hash() (string, error) {
+	// get pktoken bytes
+	pktJson, err := p.ToJSON()
```

Comment:
I worry that `pkt-->pktJson` introduces serialization differences. Ideally we would represent all content that has been signed in a compact format such that all signatures can be checked without deserialization.

Reply:
8954470 is my short term answer to this. I've added a `raw` field to our pktoken so that when we are unmarshaled from an existing json, we use whatever that was and if we're a new pktoken then we'll marshal ourselves. I personally think we're still in a stage where it's more important to match the paper (since this is essentially touted as a reference implementation) than something universally built for all languages. Hopefully, by that time we will have a new plan for compaction and we can go that route.